RELEASE_NOTES.md

```diff
@@ -2,17 +2,17 @@
 
 ## Summary
 
+<!-- Here goes a general summary of what this release is about -->
+
 ## Upgrading
 
+<!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with -->
+
 ## New Features
 
-* A new class `OrderedRingBuffer` is now available, providing a sorted ring buffer of datetime-value pairs with tracking of any values that have not yet been written.
-* Add logical meter formula for EV power.
-* A `MovingWindow` class has been added that consumes a data stream from a logical meter and updates an `OrderedRingBuffer`.
-* Add EVChargerPool implementation. It has only streaming state changes for ev chargers, now.
-* Add 3-phase current formulas: `3-phase grid_current` and `3-phase ev_charger_current` to the LogicalMeter.
 * A new class `SerializableRingbuffer` is now available, extending the `OrderedRingBuffer` class with the ability to load & dump the data to disk.
```

Comment:
@ela-kotulska-frequenz is this correct to remove all the features about and keep `SerializableRingbuffer` as it hasn't been released yet?

Reply:
No. They should stay. You should just add your features. :0

Reply:
Yes, it is fine